README.md

```diff
@@ -104,7 +133,23 @@ You're a good person and decide to validate your HTML. You copy and paste the co
 The validator throws errors at you! Why? Assuming you haven't made any mistakes in your code, how could you go about accurately validating your HTML?
 
 ```
-Your answer...
+put in your own validation:
+
```

Comment:
I love this answer, I think the question was more asking why the view code in our Rails apps is not valid html? That's because Rails uses ERB, which is embedded Ruby, and therefore not valid Ruby...In order to check your outputted html, just view the source from the browser and check that HTML